CONTRIBUTORS.md

```diff
@@ -11,6 +11,7 @@ If you have contributed and do not see your name on this list, please raise an i
 * Jessica Jones
 * Laurence Smith (@Sourceless)
```

Comment:
This list is very out of date. I'd just remove it, or the file, unless there's a specific need.

Reply:
No worries I'll raise a separate PR for that.